tools/clang/unittests/HLSL/DxilContainerTest.cpp

```diff
@@ -2084,6 +2085,115 @@ TEST_F(DxilContainerTest, ValidateFromLL_Abs2) {
   CodeGenTestCheck(L"..\\CodeGenHLSL\\container\\abs2_m.ll");
 }
 
+// Test to see if the Compiler Version (VERS) part gets added to library shaders
+// with SM >= 6.8
```

Comment:
Should change comment that says "`with SM >= 6.8`" to "`with validator version >= 1.8`"
The `m_ver.SkipDxilVersion(1, 8)` below is still correct to constrain for validator version.  However, the library target can be lower.